snapcraft/templates/kernel/ubuntu_kernel_get_build_commands.sh.j2

```diff
@@ -0,0 +1,63 @@
+# Ubuntu kernel snapcraft build script
+# Generated by snapcraft ubuntu-kernel plugin.
+# ----
+# Build on:  {{ craft_arch_build_on }}
+# Build for: {{ craft_arch_build_for }}
+# Snapcraft version: {{ snap_version }}
+# Snap data: {{ snap_data_path }}
+# Snap context: {{ snap_context }}
+# ----
+env
+
+# Cleanup build arena. Rebuilding without a `snapcraft clean` leaves state
+# behind, which slows down incremental builds significantly during development.
+cd {{ craft_part_build_dir }}
+rm -fr {{ craft_part_build_dir }}/kernel/
+rm -fr {{ craft_part_build_dir }}/unpacked-*
+rm -fr {{ craft_part_install_dir }}/*
+
+mkdir {{ craft_part_build_dir }}/kernel
```

Comment:
I understand that under `{{ craft_part_build_dir }}` the script is also creating a `{{ craft_part_build_dir }}` dir. Is that on purpose or is that a mistake?